lib/src/external_module.cc

```diff
@@ -313,8 +315,10 @@ ActionResponse ExternalModule::callBlockingAction(const ActionRequest& request)
     LOG_TRACE("Input for the %1%: %2%", request.prettyLabel(), action_args);
 
     auto exec = lth_exec::execute(
-#ifdef _WIN32
+#if defined(_WIN32)
         "cmd.exe", { "/c", path_, action_name },
+#elif defined(__sun)
+        "/usr/bin/ctrun", { "-l", "child", path_, action_name },
```

Comment:
I was considering not using ```ctrun``` here (and above), but in the end I decided to use it here to keep the behavior consistent with other OSes.